setup.py

```diff
@@ -29,9 +29,12 @@
             packages=[
                 p for p in find_packages() if not p.endswith('.test')
             ],
-            data_files=[
-                ('sbin/', [PACKAGE_NAME, 'directory-bootstrap']),
-            ],
+            entry_points={
+                'console_scripts': [
+                    'directory-bootstrap = directory_bootstrap.cli:main',
+                    'image-bootstrap = image_bootstrap.cli:main'
```

Comment:
Please add a trailing comma to that line for better diffs when we extend again, next time